src/orion/storage/track.py

```diff
@@ -275,6 +276,12 @@ def results(self):
 
         return self._results
 
+    @property
+    def hash_params(self):
+        """See `~orion.core.worker.trial.Trial`"""
+        from orion.core.worker.trial import Trial as OrionTrial
+        return OrionTrial.compute_trial_hash(self, ignore_fidelity=True)
```

Comment:
You turn the property into a staticmethod so that you can pass a duck that quacks like a duck but is not a duck. :thinking:

Reply:
PYTHONNNNNN am I rite